lang/es.xml

```diff
@@ -167,7 +183,7 @@ No apagues el sistema ni cierres la aplicación.</warning_saving>
 	</live_area>
 
 	<settings name="Ajustes">
-		<theme_background name="Tema y Fondo">
+		<theme_background name="Tema y fondo">
```

Comment:
wut ? revert

Reply:
no you no undersund is juste normal, check all languague, is same for all

Reply:
I'll revert it, but let me insist that this string is inadequate, even if it comes from SCEE.